theClanless/src/main/resources/theClanlessResources/localization/eng/DefaultMod-Keyword-Strings.json

```diff
@@ -1,68 +1,14 @@
 [
   {
-    "PROPER_NAME": "additional strike",
+    "PROPER_NAME": "Additional Strike",
```

Comment:
I figured out how it works. You can see the comments in theClanlessMod.java line 536-542.

PROPER_NAME = The name of the power in the tool tip. Case sensitive.
NAMES = Different ways of identifying the keyword. Not case sensitive, BUT in the display text it will use the case you select.